app/views/posts/_posts.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class='col-9 caja1 mx-auto caja'>    
+    <% @posts.each do |post| %>
+        
+            <div class='col-11  bg-light row mx-auto'> 
+                <% if usuario_signed_in?%>          
+                    <div class='col-2 mx-auto bg-light'>                        
+                        <div>
+                            <p class="image is-64x64 ">
+                                <img src="https://bulma.io/images/placeholders/64x64.png" class='img-container'>
+                            </p>
+                        </div>
+                            <strong><p> <%= post.usuario.name %></p></strong><br />                      
+                    </div>
+                <% end %>    
```

Comment:
Please move logic into view helpers

Reply:
????

Reply:
where i have to set  the helper???
can you tell me?

Reply:
You can put the login in the `helpers` directory
![hp](https://user-images.githubusercontent.com/46388113/93640229-c4a40080-f9e9-11ea-8405-973de9fe0c50.png)